installer/modelsDLTK.py

```diff
@@ -43,10 +43,32 @@ def addModels(login: str, modelPath: str, update: bool):
 	
 	logger(f"Trying to upload the model using {login} as login",forceConsoleOutput=True)
 	args = "%s %s %s %s" % (login, os.path.abspath(localFileModel), SCIPION_SOFTWARE_EM, update)
-	retCode, log = runJob(f"dist/bin/xmipp_sync_data upload {args}", showCommand=True, showError=True)
+	retCode, log = runJob(f"dist/bin/xmipp_sync_data upload {args}", showCommand=False, showError=True)
 	if retCode == 0:
 		logger(f"{modelName} model successfully uploaded! Removing the local .tgz")
 		runJob("rm %s" % localFileModel)
 		
 		
-		
+
+def downloadDeepLearningModels(dest:str, distPath: str):
+    if not os.path.exists('dist/bin/xmipp_sync_data'):
+        logger(red('Xmipp has not been installed. Please, first install it '))
+        sys.exit()
+    if dest == os.path.join(distPath):
+        modelsPath = os.path.join(dest, 'models')
+    else:
+        modelsPath = dest
+    dataSet = "DLmodels"
+
+    # downloading/updating the DLmodels
+    if os.path.isdir(modelsPath):
+        logger("Updating the Deep Learning models (in backgound)")
+        task = "update"
+    else:
+        logger("Downloading Deep Learning models (in backgound)")
+        task = "download"
+    retCode, log = runJob(f"dist/bin/xmipp_sync_data {task} {modelsPath} {MODELS_URL} {dataSet}",
+           showCommand=True, showError=True, showOutput=True)
+    if retCode == 0:
+        logger(f"Models successfully {task}")
```

Comment:
No error control on the command? What to do if it fails?

Reply:
The runJob would report the error message, we dont need nothing else